Dashboard/app/js/lib/models/models.js

```diff
@@ -347,6 +347,7 @@ FLOW.Placemark = FLOW.BaseModel.extend({
 FLOW.SurveyInstance = FLOW.BaseModel.extend({
   approvedFlag: DS.attr('string'),
   approximateLocationFlag: DS.attr('string'),
+  surveyInstanceId: DS.attr('number'),
```

Comment:
The `keyId` is the same as `surveyInstanceId` so perhaps we can use that instead?